Week3/homework/step4.js

```diff
@@ -1,8 +1,11 @@
 'use strict';
 
-// Add your code here
 
-const addSix = createBase(6);
 
-addSix(10); // returns 16
-addSix(21); // returns 27
+const addSix = function (number) {
+    return number + 6;
+}
+
+
+console.log(addSix(10)); // returns 16
+console.log(addSix(21)); // returns 27
```

Comment:
Yes, this fills the brief.  But I think the idea was that you created a generic solution that would allow someone to create an `addSeven` or an `addTwenty` with the same code.  Something like this:
```
const createBase = (adder) => (n) => adder + n;
```